docs/reference/eql/eql-search-api.asciidoc

```diff
@@ -61,12 +62,13 @@ See <<eql-limitations,EQL limitations>>.
 [[eql-search-api-path-params]]
 ==== {api-path-parms-title}
 
-`<index>`::
+`<target>`::
 (Required, string)
-Comma-separated list of index names or <<indices-aliases,index aliases>> used to
-limit the request. Accepts wildcard expressions.
+Comma-separated list of data streams, indices, or <<indices-aliases,index
+aliases>> used to limit the request. Accepts wildcard (`*`) expressions.
 +
-To search all indices, use `_all` or `*`.
+To search all data streams and indices in a cluster, use
+`_all` or `*`.
```

Comment:
As of this writing, a bug prevents this from resolving to data streams. See #58562.